crates/biome_js_parser/src/syntax/typescript/types.rs

```diff
@@ -1125,6 +1121,10 @@ fn parse_ts_import_type(p: &mut JsParser, context: TypeContext) -> ParsedSyntax
     p.expect(T![import]);
     p.expect(T!['(']);
     p.expect(JS_STRING_LITERAL);
+    if p.at(T![,]) {
+        p.bump(T![,]);
+        parse_ts_object_type(p, context).or_add_diagnostic(p, expected_identifier);
```

Comment:
I add a `parse_ts_object_type`  for parse `{ with: { 'resolution-mode': 'import' } }` here, but i got a `modified tree has bogus nodes or empty slots:` error here at my local, i am not sure why function `parse_ts_import_type`  cause `TS_BOGUS_TYPE` here, because  `type E = { with: { 'resolution-mode': 'import' } }` can be deal by `parse_ts_object_type` normally...

Reply:
and i'm also not sure the change for parse here is right or not...

Reply:
@Conaclos If you have some spare time, could you please offer me some advice? Thank you.~

Reply:
Hi @fireairforce, sorry for the delay! I think it is because you didn't change the grammar (in the `js.ungrammar` file). Once changed accordingly you have to run `just gen-grammar`.